{"uuid":"acf3c240-998a-4047-9edd-752794d8d71a","historyId":"8bf9408689cde93e0668e657e595ffe7","fullName":"Array: The user is presented with error message for code with invalid syntax in tryEditor for Find Numbers with Even Number of Digits","labels":[{"name":"tag","value":"arrays"},{"name":"host","value":"LAPTOP-RSES787T"},{"name":"thread","value":"48452@LAPTOP-RSES787T.main(1)"},{"name":"feature","value":"Array"},{"name":"story","value":"The user is presented with error message for code with invalid syntax in tryEditor for Find Numbers with Even Number of Digits"},{"name":"suite","value":"Array"},{"name":"testClass","value":"The user is presented with error message for code with invalid syntax in tryEditor for Find Numbers with Even Number of Digits"},{"name":"framework","value":"cucumber4jvm"},{"name":"language","value":"java"},{"name":"gherkin_uri","value":"file:///C:/Users/valli/Git/DS-ALGO/src/test/resources/features/5arrays.feature"},{"name":"package","value":"src.test.resources.features.5arrays_feature.Array"}],"links":[],"name":"The user is presented with error message for code with invalid syntax in tryEditor for Find Numbers with Even Number of Digits","status":"passed","stage":"finished","steps":[{"name":"Given  The user is on \"Question\" page of \"Find Numbers with Even Number of Digits\" after logged in","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673380838012,"stop":1673380838239},{"name":"When  The user enter python code with invalid syntax in tryEditor from sheet \"pythonCode\" and 1","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673380838239,"stop":1673380838588},{"name":"And  The user clicks on run button","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673380838588,"stop":1673380838713},{"name":"Then  The user should be presented with error message as \"SyntaxError: bad input on line 1\"","status":"passed","statusDetails":{"known":false,"muted":false,"flaky":false},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1673380838713,"stop":1673380838722}],"attachments":[],"parameters":[{"name":"Sheetname","value":"pythonCode"},{"name":"RowNumber","value":"1"}],"start":1673380838011,"stop":1673380838724}